packages/dx-react-chart-demos/src/demo-sources/export/bootstrap4/demo.jsx

```diff
@@ -0,0 +1,283 @@
+import React, { useState } from 'react';
+import {
+  ArgumentAxis,
+  ValueAxis,
+  Chart,
+  Legend,
+  BarSeries,
+  Title,
+} from '@devexpress/dx-react-chart-bootstrap4';
+import { Popover, PopoverBody } from 'reactstrap';
```

Comment:
We have custom `Popover` for *bootstrap* - can't it be used?

Reply:
Our custom `Popover` is not documented. Can we use it in our demos?

Reply:
Let these two stay then.